tools/contractdeployer/README.md

```diff
@@ -0,0 +1,15 @@
+# Contract deployer
```

Comment:
I feel like there might be a lib out there that does parts of this contract deployment stuff already (and handle things like the nonce stuff for us). But we can maybe look into that longer term if we find ourselves deploying L1 contracts more often.

Reply:
Yeah. OOS for this PR imo.